test/DynamoCoreTests/MessageLogTests.cs

```diff
@@ -40,6 +40,7 @@ public void AddWarning()
 
         protected override void GetLibrariesToPreload(List<string> libraries)
         {
+            libraries.Add("ProtoGeometry.dll");
```

Comment:
You can use FFITarget.dll - it's a ZT node test package. You can write a new test node to that assembly that logs an info message using the new API you added.

Reply:
@aparajit-pratap Never touched that, where is the code for that package? Also do you mean we cant use protoGeometry.dll in this test?

Reply:
We can, but it's better not to add dependencies to protogeometry for testing and all the overhead that comes along with it. It's just easier to use FFITarget project that's a test project in the main Dynamo solution itself. Not a big deal in this case as Protogeometry is already a dependency of DynamoCoreTests I'm assuming.